torch/csrc/Module.cpp

```diff
@@ -362,6 +371,7 @@ static PyMethodDef TorchMethods[] = {
   {"_from_dlpack",    (PyCFunction)THPModule_fromDLPack,        METH_O,       NULL},
   {"set_flush_denormal", (PyCFunction)THPModule_setFlushDenormal, METH_O,     NULL},
   {"get_default_dtype", (PyCFunction)THPModule_getDefaultDtype, METH_NOARGS,  NULL},
+  {"is_default_cuda_device", (PyCFunction)THPModule_isDefaultCudaDevice, METH_NOARGS,  NULL},
```

Comment:
can we not make this part of the public API?   Maybe _is_default_type_cuda?

Reply:
(the underscore indicating it's not part of the public api; also there's not _really_ a concept of a default device currently.